test/jdk/com/sun/jdi/FileSocketTransportTest.java

```diff
@@ -94,6 +94,16 @@ public static void main(String[] args) throws Throwable {
         }
 
         String socketName = "test.socket";
+        if (Platform.isWindows()) {
+            UnixDomainSocketAddress addr = UnixDomainSocketAddress.of(socketName);
+
+            try {
+                SocketChannel channel = SocketChannel.open(StandardProtocolFamily.UNIX);
```

Comment:
Maybe it's possible to wrap this in a try-with-resources to ensure proper closing of the channel after a successful test?